virttest/libvirt_cgroup.py

```diff
@@ -174,7 +174,13 @@ def __get_cg_file_path(cg_key, cg_path, cg_file_name):
                             dev_list.append(dev_num)
         elif virsh_cmd == "memtune":
             cgroup_path = self.get_cgroup_path("memory")
-            max_mem_value = "9223372036854771712"
+            cmd = "getconf PAGE_SIZE"
+            page_size = process.run(cmd, ignore_status=True, shell=True).stdout_text.strip()
```

Comment:
How about we also log `page_size` to output to make the assignment of max_mem more clear?